lib/features/campaigns/widgets/map_container.dart

```diff
@@ -81,10 +78,17 @@ class MapContainer extends StatefulWidget {
 }
 
 class _MapContainerState extends State<MapContainer>
-    with MapContainerExperienceAreaMixin, MapConsumerRouteMixin, MapConsumerPollingStationMixin
+    with
+        MapContainerExperienceAreaMixin,
```

Comment:
Super duper nitpicky, but I noticed this is an indent by four whereas others (like the if statement below or the class/final declaration above) has two spaces. Is that a dart convention or just a confused IDE?